Sources/Stevenson/Errors.swift

```diff
@@ -127,3 +177,13 @@ extension Future {
         }
     }
 }
+
+extension SlackResponse {
+    public init(error: Error, visibility: Visibility = .user) {
+        #if DEBUG
+        self.init(String(describing: error), visibility: visibility)
+        #else
+        self.init(error.localizedDescription, visibility: visibility)
```

Comment:
Basically, it's just about unifying error output, for some reason decoding errors behave differently than others. It's tricky though, I'm not sure what's the best option and don't like this condition because Vapor is not designed for them, they instead have production vs development environments. And there is also AbortError that is supposed to take care of it, but I didn't have time to integrate it.